src/components/SearchModal/CurrencySearch.tsx

```diff
@@ -171,6 +198,26 @@ export function CurrencySearch({
   // if no results on main list, show option to expand into inactive
   const filteredInactiveTokens: Token[] = useSearchInactiveTokenLists(debouncedQuery)
 
+  const getVisibleTokens = (): Token[] => {
```

Comment:
Why we have to write it as a function. I see we have no reason here.

Just write it as a variable with result: `const isVisibleTokens = useMemo(...)`

Reply:
I don't see it's needed a `useMemo` here

Reply:
My main point is not about useMemo. It's about why would you write it in a calculating-function, then you call it later. Instead you can calculate it right here and use result later.

Reply:
OK. I will update this with IIFE, instead of useMemo. This is not an expensive computation so a function without memo is still good.

Reply:
ýe. Usually we write it as a function only for callback. You might make ppl confuse when read this.

Reply:
It's just IIFE bro. The function executes and returns a value. it's dead simple 😅 Otherwise, give me a good reason to useMemo here

Reply:
FYI, React team quotes this in their official doc
> You may rely on useMemo as a performance optimization, not as a semantic guarantee